graph-db/src/utils/cmr/__test__/fetchPageFromCmr.test.js

```diff
@@ -0,0 +1,127 @@
+const nock = require('nock')
+
+const getEchoToken = require('../getEchoToken')
```

Comment:
`getEchoToken` isnt using a default export, it's like a nicety of JS that its importing this correctly, I think you want to update this to a named export `const { getEchoToken } = require('../getEchoToken')`

Reply:
My mocks don't seem to work when I do this as a named export